### ISSUE TITLE
Elder Scrolls Online: invisible models (AMDVLK and Windows) [Vega 56]

### ISSUE THREAD

**Issue by ssorgatem** [doitsujin/dxvk#68]
Trying out the latest build with ESO, now character models are invisible (it was all correctly rendered last week).

On the plus side, at least it doesn't hang the GPU forcing a reboot, as it used to do.

### Software information
Elder Scrolls Online, 64 bit Morrowind client, on a clean WINEPREFIX

### System information
- GPU: Vega 56
- Driver: mesa-git
- Wine version: lastest staging with some extra patches (AUR "wine-gaming-nine")
- DXVK version: latest git version as of this writing

### Apitrace file(s)
- Put a link here

### Log files
Can be found here:
https://actinia.ddns.net/index.php/s/wm3JpqQ8A7xpQPG

(eso64_dxgi.log, eso64_d3d11.log and eso64.trace; I hope to have generated them properly)

**Comment by ssorgatem** [comment]
I also tried with latest amdvlk, witht he exact same results.
It's an improvement there, since last week I couldn't get past character selection on amdvlk; although characters did render properly.

**Comment by ssorgatem** [comment]
I tried with mesa 17.3.3.

Result: the GPU freezes at the login screen, forcing a reboot.

I've attached the .log files to the same shared folder (prefixed with "mesa_stable")

**Comment by ssorgatem** [comment]
@exolyte what kernel are you running?

I'm running the latest mainline git. AMDGPU differences might be involved...

**Comment by exolyte** [comment]
I'm using the latest kernel in the arch repos: linux 4.15.2-2.

**Comment by fagoatse** [comment]
Vega based GPUs are still broken it seems.

**Comment by doitsujin** [comment]
Your trace file is broken.

Vega alone shouldn't be the reason why some of the pipelines don't compile. Can you bisect this? "Last week" is a bit vague and I can't reproduce this with the trace I have.

**Comment by ssorgatem** [comment]
OK, this is weird, I can't replicate it now on RADV.
It was probably a mesa issue... (I update mesa-git daily...)

However, it kinda shows that the freezing of the GPU is related to therendering of the character models: no characters, no freezing.

Now it renders properly.... but it freezes the GPU after some time (less than a minute).

The issue of the missing models still exists for AMDVLK, though.
So this issue can be marked AMDVLK-specific, I guess.

**Comment by ssorgatem** [comment]
For what it's worth, I've added log files for the freeze with latest dxvk  and RADV to the shared folder.

I also added a new trace file... if it's broken too I don't know what I am doing wrong :/

**Comment by ssorgatem** [comment]
In AMDVLK's case, it may be a driver issue and unrelated to dxvk. From their release notes:

"
Note: Pipeline compiler support for geometry and tessellation shaders is not fully implemented for the Radeon™ RX Vega Series. APU support is limited. These issues will be addressed in upcoming releases
"

**Comment by ssorgatem** [comment]
Just an update.

I tested it with DXVK on Windows 10 with the latest AMD driver.

Exact same behaviour as with AMDVLK on Linux. Which is not that surprising, since AMDVLK  theoretically shares the code with the Windows Vulkan driver. But it's good to confirm it's not Linux-specific.

So it's either an issue within the AMD stack or in DXVK. I'd lean to the former, given they admit they don't fully support Vega yet.

I've also updated the title of the issue.

I'll make sure to update whenever there's a new AMDVLK code drop.
